.changeset/wet-cats-allow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-home': patch
+---
+
+Added a new Featured Docs component to plugin-home, which can display any entity given a filter
```

Comment:
A quick example of the component in use here is super helpful. The changesets get rolled up into release notes so having an example here means Adopters upgrading can start using it right away. Or, you could include a link to the StoryBook entry seeing as you nicely have one :)

Reply:
done :)